### PR TITLE
Adding comment to the protobuf contract and renaming the package for …

### PR DESCRIPTION
**Pull request by danieldooley**
…consistency with package name in the gloria project

### CODE REVIEW

**Comment by danieldooley** [comment]
@gclitheroe Seems I can't set anybody to review this, must be because it's your repo.

**Comment by danieldooley** [comment]
@gclitheroe I'm committing the code for the generation of gloria protobufs. I've manually tested it and it generates protobuf files that work in Gloria. Probably needs some larger scale testing though.

File: tools/gloria/main.go [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+	"path/filepath"
+	"sort"
+	"strings"
+	"github.com/golang/protobuf/proto"
+	"encoding/json"
+)
+
+const DateTimeFormat = "2006-01-02T15:04:05"
+
+var HeaderComments map[string]string = map[string]string{
```
Comment (gclitheroe) [inline tools/gloria/main.go]: it doesn't look like this is used?  If it is not used then it may as well be removed.

File: tools/gloria/main.go [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+	if err := meta.LoadList(filepath.Join(network, "monuments.csv"), &monumentList); err != nil {
+		fmt.Fprintf(os.Stderr, "error: unable to load monuments list: %v\n", err)
+		os.Exit(-1)
+	}
+	monuments := make(map[string]meta.Monument)
+	for _, m := range monumentList {
+		monuments[m.Mark] = m
+	}
+
+	var sessionList meta.SessionList
```
Comment (gclitheroe) [inline tools/gloria/main.go]: It doesn't look like sessionList is used (and I wouldn't expect it to be).  You may as well remove the var and the type.

File: tools/gloria/main.go [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+
+	var network string
+	flag.StringVar(&network, "network", "../../network", "base network directory")
+
+	var install string
+	flag.StringVar(&install, "install", "../../install", "base install directory")
+
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "\n")
+		fmt.Fprintf(os.Stderr, "Build a RINEX configuration site XML file from delta meta information\n")
```
Comment (gclitheroe) [inline tools/gloria/main.go]: incorrect usage information here

File: tools/gloria/main.go [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+		if err := os.MkdirAll(filepath.Dir(pbfile), 0755); err != nil {
+			fmt.Fprintf(os.Stderr, "error: unable to create dir: %v\n", err)
+			os.Exit(-1)
+		}
+		if err := ioutil.WriteFile(pbfile, b, 0644); err != nil {
+			fmt.Fprintf(os.Stderr, "error: unable to write file: %v\n", err)
+			os.Exit(-1)
+		}
+
+		if m.Code == "WARK" {
```
Comment (gclitheroe) [inline tools/gloria/main.go]: is this for something specific?

**Comment by gclitheroe** [comment]
I've made a new branch and got it back in sync with upstream/master  I also cherry picked in cc78853 from this PR.  You can then make changes as suggest in my review comments and raise the new pr against https://github.com/gclitheroe/delta/tree/gloria-pb-v3

It should then all be easy to keep rebasing on upstream/master until we want to pr to that.

**Pull request closed without merging**